launch.go

```diff
@@ -121,20 +121,25 @@ func getAccountServiceURL(r *http.Request) string {
 func redirectURL(w http.ResponseWriter, r *http.Request) {
 	hostURL := settings.Get("SURVEY_RUNNER_URL")
 
-	token, err := authentication.GenerateTokenFromPost(r.PostForm)
+	launchActionv1 := r.PostForm.Get("action_launch_v1")
+    launchActionv2 := r.PostForm.Get("action_launch_v2")
+
+	tokenV1, err := authentication.GenerateTokenFromPost(r.PostForm, false)
+	tokenV2, err := authentication.GenerateTokenFromPost(r.PostForm, true)
```

Comment:
How come this is called twice with 2 vars?. We only need to generate one or the other.

Reply:
Updated now.